lib/entities/concept-scheme.ts

```diff
@@ -0,0 +1,29 @@
+import { LocalizedEntity } from './utils'
+import { TaxonomyConceptLink } from './concept'
+
+export type ConceptScheme = {
+  uri: string | null
+  prefLabel: string
+  altLabels: string[]
+  hiddenLabels: string[]
+  definition: string | null
+  notations: string[]
+  topConcepts: TaxonomyConceptLink[]
+  concepts: TaxonomyConceptLink[]
+  totalConcepts: number
+  sys: {
+    type: 'TaxonomyConceptScheme'
+    createdAt: string
+    updatedAt: string
+    id: string
+    version: number
+  }
+}
+
+export type ConceptSchemeProps<Locales extends keyof any = 'en-US'> = LocalizedEntity<
```

Comment:
with localization in mind, does it make sense to specify `'en-US'` locale? we don't know how this api could look like in the future, maybe customers will create their own default locales. maybe fallback it to just `string`?

Reply:
it does, will change it!

Reply:
fixed